src/Model/GlobalCommands.re

```diff
@@ -0,0 +1,390 @@
+
+let undo = define("undo", Command("undo"));
+let redo = define("redo", Command("redo"));
+
+let indent = define("indent", Command("indent"));
+let outdent = define("outdent", Command("outdent"));
+
+module Editor = {
+  module Action = {
+    let detectIndentation =
+      define(
+        ~category="Editor",
+        ~title="Detect Indentation from Content",
+        "editor.action.detectIndentation",
+        Command("editor.action.detectIndentation"),
+      );
+
+    let clipboardPasteAction =
+      define(
+        "editor.action.clipboardPasteAction",
+        Command("editor.action.clipboardPasteAction"),
+      );
+
+    let indentLines =
+      define(
+        "editor.action.indentLines",
+        Command("editor.action.indentLines"),
+      );
+
+    let outdentLines =
+      define(
+        "editor.action.outdentLines",
+        Command("editor.action.outdentLines"),
+      );
+  };
+};
+
+module List = {
+  let focusDown = define("list.focusDown", ListFocusDown);
+  let focusUp = define("list.focusUp", ListFocusUp);
+
+  let select = define("list.select", ListSelect);
+  let selectBackground =
+    define("list.selectBackground", ListSelectBackground);
+};
+
+module Oni = {
+  module Explorer = {
+    let toggle =
+      define(
+        ~category="View",
+        ~title="Toggle File Explorer visibility",
+        "oni.explorer.toggle", // use workbench.action.toggleSidebarVisibility instead?
+        Actions.ActivityBar(ActivityBar.FileExplorerClick),
+      );
+  };
+
+  module KeyDisplayer = {
+    let disable =
+      define(
+        ~category="Input",
+        ~title="Disable Key Displayer",
+        ~isEnabledWhen=WhenExpr.parse("oni.keyDisplayerEnabled"),
+        "oni.keyDisplayer.disable",
+        DisableKeyDisplayer,
+      );
+
+    let enable =
+      define(
+        ~category="Input",
+        ~title="Enable Key Displayer",
+        ~isEnabledWhen=WhenExpr.parse("!oni.keyDisplayerEnabled"),
+        "oni.keyDisplayer.enable",
+        EnableKeyDisplayer,
+      );
+  };
+
+  module Sneak = {
+    let start =
+      define(
+        ~category="Sneak",
+        ~title="Enter sneak mode (keyboard-accessible UI)",
+        "oni.sneak.start",
+        Command("sneak.start"),
+      );
+
+    let stop =
+      define(
+        ~category="Sneak",
+        ~title="Exit sneak mode",
+        "oni.sneak.stop",
+        Command("sneak.stop"),
+      );
+  };
+
+  module System = {
+    let addToPath =
+      define(
+        ~category="System",
+        ~title="Add Oni2 to System PATH",
+        ~isEnabledWhen=WhenExpr.parse("isMac && !symLinkExists"),
+        "oni.system.addToPath",
+        Command("system.addToPath"),
+      );
+
+    let removeFromPath =
+      define(
+        ~category="System",
+        ~title="Remove Oni2 from System PATH",
+        ~isEnabledWhen=WhenExpr.parse("isMac && symLinkExists"),
+        "oni.system.removeFromPath",
+        Command("system.removeFromPath"),
+      );
+  };
```

Comment:
cc @CrossR. I struggled to understand the logic of the original enablement condition for these, but it seems like this was the intent at least.

Reply:
Looks good to me! The original logic was a bit sloppy...